api-client/src/main/java/com/xing/api/resources/MessagesResource.java

```diff
@@ -0,0 +1,134 @@
+     * {@linkplain Conversation conversations}.
+     * @return A {@linkplain CallSpec callSpec object} ready to execute the request.
+     *
+     * @see <a href="https://dev.xing.com/docs/get/users/:user_id/conversations">'List user conversations' resource
+     * page</a>
+     */
+    public CallSpec<List<Conversation>, HttpError> getConversationsByUserId(String userId) {
+        return Resource.<List<Conversation>, HttpError>newGetSpec(api, "/v1/users/{id}/conversations")
+              .pathParam("id", userId)
+              .responseAs(list(Conversation.class, "conversations", "items"))
+              .build();
+    }
+
+    /**
+     * Starts a {@linkplain Conversation conversation} by sending the passed message to the recipients.
+     * The subject of a conversation cannot be changed afterwards.
+     * Basic members are not allowed to send messages to non-contacts.
+     * Premium members are limited to 20 messages to non-contacts per month.
+     *
+     * @param content Message text with max size of 16384 UTF-8 characters
+     * @param recipientIds Comma-separated list of recipients. There must be between one and 10 recipients.
+     * Sender cannot be included.
+     * @param subject Subject for conversation. Max. size is 255 UTF-8 characters
+     * @param userId Id of the {@linkplain com.xing.api.data.profile.XingUser user} starting the conversation
+     * @return A {@linkplain CallSpec callSpec object} ready to execute the request.
+     *
+     * @see <a href="https://dev.xing.com/docs/post/users/:user_id/conversations">'Create a conversation' resource page</a>
+     */
+    public CallSpec<Conversation, HttpError> createConversation(String content, String recipientIds, String subject,
+          String userId) {
+        return Resource.<Conversation, HttpError>newPostSpec(api, "/v1/users/{user_id}/conversations", false)
+              .pathParam("user_id", userId)
+              .queryParam("content", content)
+              .queryParam("recipient_ids", recipientIds)
+              .queryParam("subject", subject)
+              .responseAs(single(Conversation.class, "conversation"))
+              .build();
+    }
+
+    /**
+     * Creates a new {@linkplain ConversationMessage message} in an existing {@linkplain Conversation conversation}.
+     *
+     * @param messageContent The {@linkplain ConversationMessage message} to add
+     * @param conversationId Id of the {@link Conversation conversation} where the message is created in
+     * @param userId Id of the {@linkplain com.xing.api.data.profile.XingUser user} who sends the message
+     * @return A {@linkplain CallSpec callSpec object} ready to execute the request.
+     *
+     * @see <a href="https://dev.xing.com/docs/post/users/:user_id/conversations/:conversation_id/messages">'Create
+     * message in a conversation' resource page</a>
+     */
+    public CallSpec<ConversationMessage, HttpError> sendMessageToConversation(String messageContent,
```

Comment:
Same here... should be `sendMessageToConversation(userId, conversationId, messageContent)`